dockerfile_functions.sh

```diff
@@ -368,6 +368,13 @@ print_java_install_pre() {
        armhf|armv7l) \\
          ESUM='$(get_shasum "${shasums}" armv7l "${osfamily}")'; \\
          BINARY_URL='$(get_v3_binary_url "${JAVA_URL}")'; \\
+		EOI
+			if [ "${version}" == "8" ] && [ "${vm}" == "hotspot" ] && [ "${os}" == "ubuntu" ]; then
+				cat >> "$1" <<'EOI'
+         apt-get install -y --no-install-recommends libatomic1 \
+EOI
```

Comment:
Whitespace

Reply:
I originally followed the pattern from the s390/FFI check below. I'm changing this to match the more-typical usage.

Reply:
should be fixed in 0017de34.